src/main/twirl/com/lightbend/lagom/docs/getstartedjavamaven.scala.html

```diff
@@ -31,21 +31,14 @@ <h1>Using Lagom with Java and Maven</h1>
                         <li>Maven 3.6 or higher</li>
                     </ul>
                       </p>
-                      <p>The Lightbend Tech Hub Project Starter makes it simple to create your first project. It invokes the Maven <code>mvn archetype:generate</code> command to create a Lagom Hello World project and compresses it in a zip file.
+                      <p>We’ve made it easy to create your first project. The link below invokes the Maven <code>mvn archetype:generate</code> command to create a Lagom Hello World project and compresses it in a zip file.
```

Comment:
Maybe use 'apostrophe' instead of 'right single quotation mark'?

```suggestion
                      <p>We've made it easy to create your first project. The link below invokes the Maven <code>mvn archetype:generate</code> command to create a Lagom Hello World project and compresses it in a zip file.
```

Reply:
Oh wow, that's some snipper sight here. Thanks @raboof